content/06.methods.md

```diff
@@ -125,7 +124,7 @@ This workflow is available in the [D3b GitHub repository](https://github.com/d3b
 
 ##### SNV and indel calling
 
-For PBTA samples, we used four variant callers to call SNVs and indels from panel, WXS, and WGS data: `Strelka2` [@doi:10.1038/s41592-018-0051-x], `Mutect2` [@biorxiv:10.1101/861054], `Lancet` [@doi:10.1038/s42003-018-0023-9], and `VarDictJava` [@doi:10.1093/nar/gkw227].
+We used four variant callers to call SNVs and indels from paired tumor/normal samples with one or more of: Targeted Panel, WXS, and/or WGS data: `Strelka2` [@doi:10.1038/s41592-018-0051-x], `Mutect2` [@biorxiv:10.1101/861054], `Lancet` [@doi:10.1038/s42003-018-0023-9], and `VarDictJava` [@doi:10.1093/nar/gkw227].
```

Comment:
#382

Reply:
Lots of `:` here now. I'm not sure what "one or more of" means, doesn't that just mean "everything"? Would this update make sense or is there a detail I'm missing? It seems from #382 that we really need to just focus on the _paired_ aspect. Eh?
 
```suggestion
We used four variant callers to call SNVs and indels from paired tumor/normal samples with Targeted Panel, WXS, and/or WGS data: `Strelka2` [@doi:10.1038/s41592-018-0051-x], `Mutect2` [@biorxiv:10.1101/861054], `Lancet` [@doi:10.1038/s42003-018-0023-9], and `VarDictJava` [@doi:10.1093/nar/gkw227].
```

Reply:
ah yeah when I read it back now the and/or takes care of it!